core/tauri/src/app.rs

```diff
@@ -651,6 +660,15 @@ impl<R: Runtime> Builder<R> {
   }
 
   /// Defines the JS message handler callback.
+  ///
+  /// # Example
+  /// ```rust,ignore
+  /// tauri::Builder::default()
+  ///   .invoke_handler(tauri::generate_handler![
+  ///     command_1,
```

Comment:
maybe also add the commands to this example? just to show that it must use #[tauri::command]

Reply:
Good call, added it!